### PR TITLE
Improve cross compilation

### PR DESCRIPTION
**Pull request by FeldrinH**
This PR improves cross compilation of the project. Cross compiling is desireable, because on Windows IntelliJ and some other IDE-s are very flaky when trying to edit a project in the WSL filesystem.

Changes:
* Added some useful scripts to compile and run GobPie in both Window and Linux 
* Fixed issue with `vscode:prepublish` command not working on Windows
* Fixed an issue with compiled jar only bundling required native JS engine dependencies for the same OS, making it not fully cross-platform.

### CODE REVIEW

File: build_and_test.bat [hunk 1]
```diff
@@ -0,0 +1,5 @@
+call mvn package
```
Comment (karoliineh) [inline build_and_test.bat:1]:
```suggestion
call mvn clean package
```

File: build_and_test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+mvn package
```
Comment (karoliineh) [inline build_and_test.sh:1]:
```suggestion
mvn clean package
```

File: install_and_test.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+eval $(opam env --switch=/home/user/goblint/analyzer --set-switch)
```
Comment (karoliineh) [inline install_and_test.sh:1]: The switch path is awfully specific. Should have at least a comment `# Replace with your ..` like below. But that still makes tracking the scripts with git painful, as everyone has their own path there.

File: build_and_test.sh (hunk 2 above)
Comment (karoliineh) [reply build_and_test.sh:1]: This fixes most of the Maven warnings about overlapping classes, but there is still one remaining that is caused by the added `graphviz-java-all-j2v8` dependency.

**Review by karoliineh — CHANGES_REQUESTED** [review]
Scripts seem to work, but I am not sure if the project root is the best place for them. As they are still VScode specific, maybe move them into the `vscode` folder where the `installGobpie.sh` script is located as well?

**Comment by FeldrinH** [comment]
I tried to move the environment-specific paths into a .env file. It works on my computer, but I am not very good with shell scripting, so it might do something weird with different setups.

**Comment by karoliineh** [comment]
> I tried to move the environment-specific paths into a .env file. It works on my computer, but I am not very good with shell scripting, so it might do something weird with different setups.

Works for me, though.

**Review by karoliineh — COMMENTED** [review]
Please also add the script files to `.vscodeignore` so that they are not bundled with the extension.

**Comment by FeldrinH** [comment]
Done. I added all .sh and .bat scripts to .vscodeignore

**Review by karoliineh — APPROVED** [review]

**Review by sim642 — APPROVED** [review]

**Pull request merged**